adapters/lora-vertx/src/main/java/org/eclipse/hono/adapter/lora/Location.java

```diff
@@ -0,0 +1,79 @@
+/**
+ * Copyright (c) 2020 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+
+
+package org.eclipse.hono.adapter.lora;
+
+import java.util.Objects;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * A 3D geo-location that consists of longitude, latitude and (optional) altitude.
```

Comment:
I think when it comes to coordinates, there should be a very explicit definition of what the coordinate system is.